db/people.yml

```diff
@@ -239,7 +239,7 @@ bfrandsen:
 bozer:
   active: true
   aka: []
-  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  avatar: https://github.com/berrakozer.png
   bio: bio
```

Comment:
I can merge this if you want to do it on a different PR, but could you add some words here in the Bio?

Reply:
@sbillinge I prepared the bio, should I add it under this PR by making changes on Pycharm?